src/cup1d/data/data_fid_eBOSS_mock.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+import pandas
+import numpy as np
+
+from cup1d.data.base_p1d_data import BaseDataP1D, _drop_zbins
+
+
+class P1D_eBOSS_mock(BaseDataP1D):
+    def __init__(
+        self,
+        diag_cov=False,
+        kmax_kms=0.09,
+        zmin=2.19,
```

Comment:
Why do you have these defaults? Are they needed? If so, I would chose more sensible values (0.09 is very high-k for an eBOSS measurement). Can we not set them to None?

Reply:
No need at all. I just copied the values from another file and forgot to set these to None, sorry!

Reply:
fixed